FlowCrypt/Functionality/Services/Local Private Key Services/PassPhraseService.swift

```diff
@@ -72,18 +72,15 @@ protocol PassPhraseServiceType {
 final class PassPhraseService: PassPhraseServiceType {
     private lazy var logger = Logger.nested(Self.self)
 
-    let currentUserEmail: String?
     let encryptedStorage: PassPhraseStorageType
     let inMemoryStorage: PassPhraseStorageType
 
     init(
-        encryptedStorage: PassPhraseStorageType = EncryptedStorage(),
-        localStorage: PassPhraseStorageType = InMemoryPassPhraseStorage(),
-        emailProvider: EmailProviderType = DataService.shared
+        encryptedStorage: PassPhraseStorageType,
+        localStorage: PassPhraseStorageType = InMemoryPassPhraseStorage()
     ) {
```

Comment:
This naming is extremely confusing an we'll need to file a separate issue just for this